### PR TITLE
Add dnu restore --skip-restore-events

### PR DESCRIPTION
**Pull request by ChengTian**
parent https://github.com/aspnet/dnx/issues/1874

@muratg @DamianEdwards @davidfowl

### CODE REVIEW

File: src/Microsoft.Framework.PackageManager/Program.cs [hunk 1]
```diff
@@ -69,6 +69,9 @@ public int Main(string[] args)
                 var optUnlock = c.Option("--unlock",
                     "Creates dependencies file with locked property set to false. Overwrites file if it exists.",
                     CommandOptionType.NoValue);
+                var optSkipRestoreEvents = c.Option("--skip-restore-events",
+                    "Suppress the execution of any scripts defined in project.json",
```
Comment (muratg) [inline src/Microsoft.Framework.PackageManager/Program.cs]: @DamianEdwards What should the help look like for this?

Comment (muratg) [inline src/Microsoft.Framework.PackageManager/Program.cs]: Remove "any" from "Suppress the execution of any scripts defined in project.json" and :shipit:

**Comment by jrieken** [comment]
:dancers:

**Pull request merged**